tornjak-backend/api/agent/types.go

```diff
@@ -20,7 +20,33 @@ type TornjakConfig struct {
 	Plugins *catalog.HCLPluginConfigMap `hcl:"plugins"`
 }
 
+/* Server configuration*/
+
 type serverConfig struct {
+	SPIRESocket	string		`hcl:"spire_socket_path"`
+	HttpConfig	*httpConfig	`hcl:"http"`
+	TlsConfig 	*tlsConfig	`hcl:"tls"`
+	MtlsConfig	*mtlsConfig	`hcl:"mtls"`
+}
+
+type httpConfig struct {
+	Enabled		bool	`hcl:"enabled"`
+	ListenPort	string	`hcl:"listen_port"`
```

Comment:
I think the current name is `listen_addr` but since we are changing to `port` then we can make it an integer and capture parsing errors in case the non-number is provided.